boards.txt

```diff
@@ -2119,6 +2119,94 @@ heltec_wifi_lora_32.menu.UploadSpeed.512000.upload.speed=512000
 
 ##############################################################
 
+heltec_wifi_lora_32_V2.name=Heltec_WIFI_LoRa_32_V2
+
+heltec_wifi_lora_32_V2.upload.tool=esptool_py
+heltec_wifi_lora_32_V2.upload.maximum_size=1310720
+heltec_wifi_lora_32_V2.upload.maximum_data_size=327680
+heltec_wifi_lora_32_V2.upload.wait_for_upload_port=true
+
+heltec_wifi_lora_32_V2.serial.disableDTR=true
+heltec_wifi_lora_32_V2.serial.disableRTS=true
+
+heltec_wifi_lora_32_V2.build.mcu=esp32
+heltec_wifi_lora_32_V2.build.core=esp32
+heltec_wifi_lora_32_V2.build.variant=heltec_wifi_lora_32_V2
+heltec_wifi_lora_32_V2.build.board=Heltec_WIFI_LoRa_32_V2
+
+heltec_wifi_lora_32_V2.build.f_cpu=240000000L
+heltec_wifi_lora_32_V2.build.flash_mode=dio
+heltec_wifi_lora_32_V2.build.flash_size=4MB
+heltec_wifi_lora_32_V2.build.boot=dio
+heltec_wifi_lora_32_V2.build.partitions=default
```

Comment:
The default LoRa32 v2 has 8MB of flash and needs a different partition map.  This can be found in PR #2407